### ISSUE TITLE
[NEWS] Wysyłam z Allegro - umożliwiamy dodanie opisu na etykiecie dla każdej paczki / Ship with Allegro - we allow you to add text on the label for each package

### ISSUE THREAD

**Issue by MaciejFrackowiak** [allegro/allegro-api#9974]
Usługa wielopaczki pozwala na przygotowanie jednej przesyłki składającej z kilku paczek, dla których generowane są osobne etykiety. Wszystkie paczki w takiej przesyłce są traktowane jako jedno zlecenie. To znaczy, że będą odebrane i dostarczone razem. Usługa obecnie jest dostępna dla przewoźników DPD i WeDo.

**Jak to działa obecnie?**

Tworząc nową przesyłkę z wykorzystaniem [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment) możesz w polu “**input.description**” dodać opis zawartości przesyłki, który pojawi się na etykiecie każdej paczki wchodzącej w skład przesyłki. Oznacza to, że każda etykieta posiada dokładnie ten sam opis.

**Co zmieniliśmy?**

Dla [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment) dodaliśmy nowe pole “**input.packages[].textOnLabel**”, w którym możesz dodać opis na etykiecie dla każdej paczki osobno wchodzącej w skład danej przesyłki.

Jednocześnie dotychczasowe pole “**input.description**” oznaczyliśmy jako **deprecated**, a w dniu **31.01.2025** całkowicie je **usuniemy**.

**Jak to będzie działało w okresie przejściowym?**

Aby zachować kompatybilność do dnia usunięcia pola “**input.description**” mechanizm będzie działał następująco:

- jeżeli przekażesz wartość w dotychczasowym polu “**input.description**” to **automatycznie skopiujemy** ją do pola “**textOnLabel**” dla każdej paczki w ramach jednej przesyłki;
- jeżeli podasz wartość w nowym polu “**textOnLabel**” to przekazana wartość w polu “**input.description**” będzie przez nas **ignorowana**.

**Dlaczego wprowadziliśmy zmianę?**

Dzięki wprowadzonej zmianie umożliwiamy precyzyjne opisanie zawartości kolejnych paczek przygotowanych w ramach jednej przesyłki.

-----
The multi-parcel service allows you to prepare one shipment of several parcels for which separate labels are generated. All parcels in such a shipment are treated as one order. That is, they will be picked up and delivered together. The service is currently available for DPD and WeDo carriers.

**How does it work now?**

When creating a new shipment using [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment), you can add a description of the shipment's contents in the “**input.description**” field, which will appear on the label of each package included in the shipment. This means that each label has the same description.

**What have we changed?**

For [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment) we have added a new field “**input.packages[].textOnLabel**”, where you can add a description on the label for each package separately included in a given shipment.

At the same time, we have marked the existing “**input.description**” field as **deprecated** and will **remove** it completely on **January 31, 2025**.

**How will it work during the transition period?**

To maintain compatibility until the day the “**input.description**” field is removed, the mechanism will work as follows:

- if you pass a value in the existing “**input.description**” field, we will **automatically copy** it to the “**textOnLabel**” field for each package within a single shipment;
- if you provide a value in the new “**textOnLabel**” field, the passed value in the “**input.description**” field will be **ignored** by us.

**Why have we made the change?**

With the change, we make it possible to accurately describe the contents of consecutive packages prepared as part of a single shipment.

**Comment by bartekc7** [comment]
Witam, czy informacja z pola textOnLabel pojawi się na etykietach innych kurierów niż DPD i WeDo?
Czy jest gdzieś dokładna informacja u jakich kurierów jest to obsługiwane?

**Comment by MaciejFrackowiak** [comment]
@bartekc7 Tak, ponieważ pole "textOnLabel" zastąpiło pole "description", w którym dotychczas można zamieścić opis przesyłki.
